index.html

```diff
@@ -0,0 +1,243 @@
+        <div class="container-fluid">
+            <button class="navbar-toggler mb-2" type="button" data-bs-toggle="collapse" data-bs-target="#collapsibleNavbar" >
+                <span class="navbar-toggler-icon" id="menu"></span>
+                <span class="fs-1 noVisible" id="cross">&cross;</span>
+              </button>
+          <!--<a class="navbar-brand noVisible fs-1" href="#" id="cross">&cross;</a>-->
+          <div class="collapse navbar-collapse" id="collapsibleNavbar">
+            <ul class="navbar-nav" id="descendant">
+                <li class="nav-item" id="homeVer" data-bs-toggle="collapse" data-bs-target="#collapsibleNavbar">
+                  <a class="nav-link" href="index.html">Home</a>
+                </li>
+                
+                <li class="nav-item" id="aboutVer" >
+                  <a class="nav-link" href="about.html">About</a>
+                </li>    
+              </ul>
+            
+          </div>
+        </div>
+    </nav>
+  </header>
+  <main>
+    <div class="accueil" id="home">
+      <h2 class="pt-5 ms-3 rouge margeB">"Hello everone, welcome to :"<br></h2>
+      <h1 class="ms-3 mb-4 rouge display-4 margeB">Open house BICC 2022</h1>
+      <div class="pWidth">
+        <p class="text-start p-3 me-3 border border-white border-2 ms-3">
+            On this wonderful day, dear guests, you will have the opportunity to discover all our works with explanation
+          </p>
+      </div>
+      <div class="margeB">
+        <p class="h2 ms-3">2023.10.15(THU) ~ 16(FRI)</p>
+        <p class="mb-5 ms-3 me-3">@Architectural design, sizing and study and more</p>
+      
+      </div>
+      <div class="container-fluid noir">
+        <h2 class="text-center text-white noir pt-5">Main Program</h2>
+        <div class="pb-5 bg-transparent"><hr class="hrrouge"></div>
+        <div class="row noir ps-3 pe-3 mobile">
+            <div class="col-md-2 grytransparent mt-2 mb-2">
+                <div class="row bg-transparent">
+                    <span class="col-2 material-icons text-white hauteur bg-transparent fs-1">
+                        work_history
+                    </span>
+                  <h4 class="col-3 hauteur rouge bg-transparent"> History</h4>
+                  <p class="col-7 bg-transparent gry">Listen to the message from the general manager who will talk about the history of their company</p>
+                </div>
+            </div>
+            <div class="col-md-2 grytransparent mt-2 mb-2">
+              <div class="row bg-transparent">
+                <!--<i class="icofont-engineer col-2 hauteur bg-transparent text-white fs-1"></i>
+                <span class="material-symbols-outlined text-white hauteur bg-transparent fs-1 col-2">
+                  engineering
+                  </span>-->
+                  <span class="material-icons text-white hauteur bg-transparent fs-1 col-2">
```

Comment:
- [ ] `required`
- [ ] Please the original background just the origin template
- [ ] please see screenshots below for further verification.

`your template`
![image](https://user-images.githubusercontent.com/66230306/217949822-146f0797-aaaf-40ed-90cb-cc1c4fca7052.png)

`original template`
![image](https://user-images.githubusercontent.com/66230306/217949963-f4997495-458d-43b5-989e-f0a2848c0244.png)